src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -245,6 +245,30 @@ def test_resource_id_scenario(self, resource_group):
         self.cmd('resource delete --id {vnet_id}', checks=self.is_empty())
 
 
+class ResourcePatchTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_resource_patch_')
+    def test_resource_patch(self, resource_group):
+        self.kwargs.update({
+            'vm': 'vm'
+        })
+        self.kwargs['vm_id'] = self.cmd(
+            'vm create -g {rg} -n {vm} --image UbuntuLTS --size Standard_D2s_v3 --v-cpus-available 1 '
+            '--v-cpus-per-core 1 --admin-username vmtest --generate-ssh-keys --nsg-rule NONE'
+        ).get_output_in_json()['id']
+
+        self.cmd(
+            'resource patch --id {vm_id} --is-full-object --properties "{{\\"identity\\":{{\\"type\\":\\"SystemAssigned\\"}}}}"',
```

Comment:
Could we update a property in the `properties` during testing? Such as: `properties.osProfile.allowExtensionOperations`